exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -188,7 +188,8 @@ public bool WasCorrectlyAnswered()
                 if (isGettingOutOfPenaltyBox)
                 {
                     Console.WriteLine("Answer was correct!!!!");
-                    playerPurseTotalState[currentPlayer]++;
+
+                    playerPurseTotalState[currentPlayer] = IncreasePlayerPurseTotal(playerPurseTotalState[currentPlayer]);
```

Comment:
I like this idea too because I can see how you're making it easier to read. But potentially adding more complexity to your code than needed. You would have to find and read the method to know what you're increasing the purse total by. Could possibly just change method name to "IncreasePlayerPurseTotalByOne".